pyproject.toml

```diff
@@ -126,6 +126,7 @@ pyshacl = { version = "^0.25.0", optional = true }
 black = { version=">=24.0.0", optional = true }
 typing-extensions = { version=">=4.4.0", python="<3.9" }
 numpydantic = { version=">=1.6.1", python = ">=3.9", optional = true}
+strenum = { version="^0.4.15", python = "<3.11" }
```

Comment:
This package is literally this with some fancy convenience methods if we wanted to avoid another dep :)

```python
from enum import Enum

class StrEnum(str, Enum): ...
```

Reply:
You are right, but that package was not added until 3.11 😄 Therefore we need a replacement for those Python "pre-historical" versions.

See [here](https://docs.python.org/3/library/enum.html#enum.StrEnum) (bottom of the picture):
<img width="473" alt="image" src="https://github.com/user-attachments/assets/663aa0b4-e95b-4ab2-bd0f-7695700f7c2f">

Reply:
What im saying is before 3.11 you can just do 

```python
class StrEnum(str, Enum): ...
```

To mimic the behavior of enum.StrEnum

Reply:
My bad. I thought you were insisting on using the `StrEnum` from the standard library from 3.11 on. Now I got it.

Reply:
Should be fixed now.